components/policies/PolicyForm.js

```diff
@@ -102,6 +104,11 @@ const PolicyForm = ({
 
     setLoading(false);
 
+    if (response.status === StatusCodes.FORBIDDEN) {
```

Comment:
Opened an issue to address the redundancy of these checks: https://github.com/rode/rode-ui/issues/165